arc/aws/391835788720/us-east-1/02_helm/variables.tf

```diff
@@ -9,3 +9,20 @@ variable "argocd_namespace" {
   type        = string
   default     = "argocd"
 }
+
+variable "cert_manager_email" {
+  type        = string
+  description = "Email to be used for the cert-manager cluster issuer"
+}
+
+variable "letsencrypt_issuer" {
+  type        = string
+  description = "Name of the cert-manager cluster issuer"
+  default     = "letsencrypt-prod"
+}
+
+variable "argocd_ingress_host" {
+  type        = string
+  description = "Public ArgoCD endpoint"
+  default     = "argocd.pytorch.org"
```

Comment:
@zxiiro This is the name that will need a new DNS record. Once this is merged and applied, I can get the IP that should go in the record. Please let me know if the name is ok - it can also be changed later but it's easier if we stick to the same name once configured.

Reply:
Name looks fine to me. Let's go with this one.